docker-compose.server.yml

```diff
@@ -10,7 +10,7 @@ networks:
 
 services:
   phpfpm:
-    image: itkdev/php8.1-fpm:alpine
+    image: itkdev/php8.2-fpm:alpine
```

Comment:
You already have something to look forward to: https://www.php.net/releases/8.3/en.php